### DISCUSSION TITLE
Use the browser add-on to view an aliases existing contacts

### DISCUSSION THREAD

**Discussion by V4ler1an** [simple-login/app#1930] [Feature Requests]
One lacking feature really frustrates me. In my day to day use of the SimpleLogin browser add-on (Firefox) I can do almost everything I need there. Create new aliases and contacts, create new contacts for an alias etc. But what I currently cannot do is look what already existing contacts belong to an alias. This frustrates me no-end because every time I want to send a new email from an alias I need to check what the contact's reverse alias is. This one requirements means I have to go and open the full SimpleLogin web site, login and search there.

In the browser add-on can you please add the ability to:

1. View a list of contacts that already belong to an alias
2. Copy / paste the existing contact's reverse alias direct from the browser add-on

Thank you!

**Comment by winklerrr** [comment]
Any news on this?
My post is related: #2426
